### PR TITLE
Update windows-enrollment-status.md

### PR DESCRIPTION
**Pull request by sdabrai**
Received feedback from customers that documentation makes it appear that default ESP profile will always win over user assigned ESP since it is assigned to "all devices" by default.

### CODE REVIEW

File: memdocs/intune/enrollment/windows-enrollment-status.md [hunk 1]
```diff
@@ -147,8 +147,8 @@ If you assign a user or device more than one ESP profile, the profile with the h
 
  Intune applies profiles in the following order:  
 
-1. Intune applies the highest-priority profile assigned to the device. 
-2. If no profiles are targeted at the device, Intune applies the highest-priority profile assigned to the user. This only works in scenarios where there is a user. In white glove and self-deploying scenarios, only profiles targeted at devices can be applied.   
+1. Intune applies the highest-priority profile assigned to the device (exluding the default ESP profile).
```
Comment (JohanFreelancer9) [inline memdocs/intune/enrollment/windows-enrollment-status.md:150]:
```suggestion
1. Intune applies the highest-priority profile assigned to the device (excluding the default ESP profile). 
```

File: memdocs/intune/enrollment/windows-enrollment-status.md [hunk 2]
```diff
@@ -147,8 +147,8 @@ If you assign a user or device more than one ESP profile, the profile with the h
 
  Intune applies profiles in the following order:  
 
-1. Intune applies the highest-priority profile assigned to the device. 
-2. If no profiles are targeted at the device, Intune applies the highest-priority profile assigned to the user. This only works in scenarios where there is a user. In white glove and self-deploying scenarios, only profiles targeted at devices can be applied.   
+1. Intune applies the highest-priority profile assigned to the device (exluding the default ESP profile). 
+2. If no profiles are targeted at the device (exluding the default ESP profile), Intune applies the highest-priority profile assigned to the user. This only works in scenarios where there is a user. In white glove and self-deploying scenarios, only profiles targeted at devices can be applied.
```
Comment (JohanFreelancer9) [inline memdocs/intune/enrollment/windows-enrollment-status.md:151]:
```suggestion
2. If no profiles are targeted at the device (excluding the default ESP profile), Intune applies the highest-priority profile assigned to the user. This only works in scenarios where there is a user. Only profiles targeted at devices can be applied in white glove and self-deploying scenarios.   
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @sdabrai, kindly commit the suggested changes. Many thanks.

File: memdocs/intune/enrollment/windows-enrollment-status.md [hunk 3]
```diff
@@ -147,8 +147,8 @@ If you assign a user or device more than one ESP profile, the profile with the h
 
  Intune applies profiles in the following order:  
 
-1. Intune applies the highest-priority profile assigned to the device. 
-2. If no profiles are targeted at the device, Intune applies the highest-priority profile assigned to the user. This only works in scenarios where there is a user. In white glove and self-deploying scenarios, only profiles targeted at devices can be applied.   
+1. Intune applies the highest-priority profile assigned to the device (excluding the default ESP profile). 
+2. If no profiles are targeted at the device (excluding the default ESP profile), Intune applies the highest-priority profile assigned to the user. This only works in scenarios where there is a user. Only profiles targeted at devices can be applied in white glove and self-deploying scenarios.
```
Comment (JohanFreelancer9) [inline memdocs/intune/enrollment/windows-enrollment-status.md:151]:
```suggestion
2. If no profiles are targeted at the device (excluding the default ESP profile), Intune applies the highest-priority profile assigned to the user. This only works in scenarios where there is a user. Only profiles targeted at devices can be applied in pre-provisioning and self-deploying scenarios.   
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @sdabrai, one more change, please. We do not use the term "white glove" in Microsoft documentation. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by lenewsad** [comment]
Hi @sdabrai, thanks for your contribution to the Microsoft Intune docs! A few build errors popped up so I incorporated your feedback in a new pull request. Changes will be live on our Learn website shortly.

**Comment by sdabrai** [comment]
[like]  Siddharth Dabrai reacted to your message:
________________________________

**Pull request closed without merging**